browser.js

```diff
@@ -34,20 +14,33 @@ function checkParameters (iterations, keylen) {
     throw new TypeError('Key length not a number')
   }
 
-  if (keylen < 0 || keylen > MAX_ALLOC) {
+  if (keylen < 0 || isNaN(keylen) || keylen > MAX_ALLOC) {
```

Comment:
probably want to do `keylen !== keylen` as older browsers aren't going to have isNaN